scripts/superdesk-authoring/views/change-image.html

```diff
@@ -16,6 +16,10 @@ <h3 class="pull-left">{{ activity.label|translate }}</h3>
                 <input type="text" ng-model="data.metadata.description_text" ng-change="data.isDirty = true" class="line-input">
             </div>
             <div>
+                <label translate>Alt Text</label>
```

Comment:
(1) I will move it one place below.
(2) We only need this `alt_text` in the context of a story for now.
Anyway, this image edit view needs to be fully upgraded since the new cropping feature (https://dev.sourcefabric.org/browse/SD-4211)